industry_benchmarks/utils/extras/plot_results.py

```diff
@@ -226,6 +226,88 @@ def plot_schrodinger_comparison(
     )
 
 
+def save_dg_values(
+        femap: FEMap,
+        filename: str,
+) -> None:
+    """
+    Helper method to write out MLE derived dG values.
+
+    Parameters
+    ----------
+    femap : FEMap
+      The cinnabar FEMap to plot
+    filename : str
+      The name of the tsv file.
+    """
+    graph = femap.to_legacy_graph()
+    writer = csv.writer(
+        filename,
+        delimiter="\t",
+        lineterminator="\n",
+    )
+    writer.writerow(["ligand", "DG (kcal/mol)", "uncertainty (kcal/mol)"])
+    # ToDo: Should the values be shifted?
+    for node in graph.nodes(data=True):
```

Comment:
Here I wasn't sure whether to save the raw values, or the "shifted" DG values.

Reply:
I would go with shifted since that better represents what we're plotting.